### PR TITLE
CSS filter function: contrast()

### PR DESCRIPTION
**Pull request by estelle**
Includes 3 new examples.

### CODE REVIEW

File: files/en-us/web/css/filter-function/contrast/index.md [hunk 1]
```diff
@@ -43,14 +197,14 @@ contrast(200%)  /* Double contrast */
...
-- {{cssxref("filter-function/blur", "blur()")}}
-- {{cssxref("filter-function/brightness", "brightness()")}}
-- {{cssxref("filter-function/drop-shadow", "drop-shadow()")}}
-- {{cssxref("filter-function/grayscale", "grayscale()")}}
-- {{cssxref("filter-function/hue-rotate", "hue-rotate()")}}
-- {{cssxref("filter-function/invert", "invert()")}}
-- {{cssxref("filter-function/opacity", "opacity()")}}
-- {{cssxref("filter-function/saturate", "saturate()")}}
-- {{cssxref("filter-function/sepia", "sepia()")}}
+- [CSS filter effects](/en-us/docs/Web/CSS/filter_effects/) module
```
Comment (teoli2003) [inline files/en-us/web/css/filter-function/contrast/index.md:200]: Fixing capitalization of slug (removing the flaw)
```suggestion
- [CSS filter effects](/en-US/docs/Web/CSS/Filter_Effects/) module
```

File: files/en-us/web/css/filter-function/contrast/index.md [hunk 2]
```diff
@@ -20,9 +20,7 @@ contrast(amount)
 ### Parameters
 
 - `amount`
-  - : The contrast of the result, specified as a {{cssxref("&lt;number&gt;")}} or a {{cssxref("&lt;percentage&gt;")}}. A value under `100%` decreases the contrast, while a value over `100%` increases it. A value of `0%` will create an image that is completely gray, while a value of `100%` leaves the input unchanged. The initial value for {{Glossary("interpolation")}} is `1`.
-
-## Examples
+  - : The contrast of the result, specified as a {{cssxref("&lt;number&gt;")}} or a {{cssxref("&lt;percentage&gt;")}}. A value under `100%` decreases the contrast, while a value over `100%` increases it. A value of `0` or `0%` will create an image that is completely gray, while a value of `1` or `100%` leaves the input unchanged. Negative values are not allowed. The initial value for {{Glossary("interpolation")}} is `1`.
 
 ### Setting contrast using numbers and percentages
```
Comment (teoli2003) [inline files/en-us/web/css/filter-function/contrast/index.md:25]: This should be under Examples

**Review by teoli2003 — CHANGES_REQUESTED** [review]
This looks good. Two small changes.

File: files/en-us/web/css/filter-function/contrast/index.md (hunk 2 above)
Comment (estelle) [reply files/en-us/web/css/filter-function/contrast/index.md:25]: I dont' think so. I didn't edit that part, but it's definitely explaining the parameter, not the examples that follow.

Comment (estelle) [reply files/en-us/web/css/filter-function/contrast/index.md]: i changed the parameter to values and changed the examples to pairs of values.

File: files/en-us/web/css/filter-function/contrast/index.md [hunk 3]
```diff
@@ -43,14 +204,14 @@ contrast(200%)  /* Double contrast */
...
-- {{cssxref("filter-function/blur", "blur()")}}
-- {{cssxref("filter-function/brightness", "brightness()")}}
-- {{cssxref("filter-function/drop-shadow", "drop-shadow()")}}
-- {{cssxref("filter-function/grayscale", "grayscale()")}}
-- {{cssxref("filter-function/hue-rotate", "hue-rotate()")}}
-- {{cssxref("filter-function/invert", "invert()")}}
-- {{cssxref("filter-function/opacity", "opacity()")}}
-- {{cssxref("filter-function/saturate", "saturate()")}}
-- {{cssxref("filter-function/sepia", "sepia()")}}
+- [CSS filter effects](/en-US/docs/Web/CSS/Filter_Effects/) module
```
Comment (teoli2003) [inline files/en-us/web/css/filter-function/contrast/index.md:207]:
```suggestion
- [CSS filter effects](/en-US/docs/Web/CSS/Filter_Effects) module
```

**Review by teoli2003 — APPROVED** [review]
👍

**Pull request merged**